metadata/src/test/java/org/apache/kafka/image/ClusterImageTest.java

```diff
@@ -17,6 +17,11 @@
 
 package org.apache.kafka.image;
 
+import org.apache.kafka.common.metadata.RegisterBrokerRecord;
```

Comment:
Can we move these imports with the other `org.apache.kafka.common.metadata` imports below?